_posts/0000-01-02-surazkarn.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
“You never change things by fighting the existing reality.
To change something, build a new model that makes the existing model obsolete.”
― Buckminster Fuller